### PR TITLE
Add multiselect date for generator

### PR DESCRIPTION
**Pull request by RobPando**

### CODE REVIEW

File: app/assets/javascripts/components/datepicker.js [hunk 1]
```diff
@@ -0,0 +1,12 @@
+$(document).ready(function() {
+  $('#multi-datepicker').datepicker({ multidate: true })
```
Comment (pablocrivella) [inline app/assets/javascripts/components/datepicker.js]: Lets use data attributes instead of css ids when we want to add JS functionality to an element. We can call this `data-datepicker`

File: app/assets/javascripts/components/datepicker.js [hunk 2]
```diff
@@ -0,0 +1,12 @@
...
+  $('#multi-datepicker').on('changeDate', function(event) {
+    $('#batch-dates').val(event.dates.join(', '))
```
Comment (pablocrivella) [inline app/assets/javascripts/components/datepicker.js]: Maybe call this something more generic in case we want to re use this component. Like just `dates` maybe?

File: app/views/meetings/_form.html.slim [hunk 3]
```diff
@@ -7,10 +7,8 @@
       = yield(:title)
 
     = form_for [@fraternity, @meeting] do |f|
-      = f.field_wrapper :date, class: 'form-group input-group' do
-        .input-group-addon
-          = icon('calendar')
-        = f.date_field :date, class: 'form-control', placeholder: t('.date')
+      = f.field_wrapper :date, class: 'form-group input-group d-flex justify-content-center', id: 'datepicker' do
```
Comment (pablocrivella) [inline app/views/meetings/_form.html.slim]: Why did you delete the `.input-group-addon` div?

Comment (RobPando) [reply app/views/meetings/_form.html.slim]: Because the inline date picker works with a hidden field as you can see so there is no need for this anymore

**Comment by RobPando** [comment]
@pablocrivella updated!
